Velocity_Model/z.py

```diff
@@ -0,0 +1,173 @@
+"""
+Library functions for extracting Z values from the NZVM in a pythonic manner
+Supports NZVM versions 2.03, 2.04, 2.06
+"""
+import os
+from pathlib import Path
+import shutil
+import subprocess
+import tempfile
+
+import numpy as np
+import pandas as pd
+from matplotlib import path as mpltPath
+
+OUT_BASIN_SIGMA = 0.5
+
+IN_BASIN_SIGMA = 0.3
+
+file_output_format = {"Z1.0": "Z/Z_1.0.txt", "Z2.5": "Z/Z_2.5.txt"}
+column_names = {"Z1.0": "Z_1.0(km)", "Z2.5": "Z_2.5(km)"}
+
+
+def get_basin_full_path(basin_list):
+    vm_dir = os.path.abspath(
+        os.path.join(os.path.dirname(os.path.realpath(__file__)), "../")
```

Comment:
Final / doesn't need to be there? And having an explicit separator defeats the purpose of using join?